cmd/registrytoken/install.go

```diff
@@ -0,0 +1,56 @@
+package registrytoken
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/cli/cli/config"
+	contextCMD "github.com/okteto/okteto/cmd/context"
+	oktetoLog "github.com/okteto/okteto/pkg/log"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type InstallOptions struct {
+	Overwrite bool
+}
+
+func Install(ctx context.Context) *cobra.Command {
+	options := &InstallOptions{}
+	cmd := &cobra.Command{
+		Use:   "install",
+		Short: "Install Okteto's docker-credential-helper",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := contextCMD.NewContextCommand().Run(ctx, &contextCMD.ContextOptions{}); err != nil {
+				return err
+			}
+			confDir := config.Dir()
+			conf, err := config.Load(confDir)
+			if err != nil {
+				return errors.Wrapf(err, "couldn't load docker config file from %q", confDir)
```

Comment:
Coiuld we use `fmt.Errorf()`. The library used is deprecated since 2.13 afaik: https://github.com/pkg/errors

Reply:
I don't think the library is deprecated, it just feature complete and locked due to the go2 errors proposal:

- https://github.com/pkg/errors#roadmap
- https://go.googlesource.com/proposal/+/master/design/go2draft.md